test/heading_increment-title_level3_ko.md

```diff
@@ -0,0 +1,5 @@
+---
+title: heading_increment-title_level3_ko
```

Comment:
"ko" => "ok", I think? :)

Reply:
Err, no, I named it "ko" on purpose, meaning that this document is wrong and raises the MD001 error, as opposed to heading_increment-title_level2_ok which is ok because it does not raise it. 
If you don't agree with the idea of having "ko" in these names, the "ok" is meaningless as well in heading_increment-title_level2_ok, and I can just remove them all.
Please let me know what you prefer.

Reply:
Instead of "ko" as a suffix, how about "good" and "bad" as used by some other test files?

Reply:
ok, done.